www-client/seamonkey/seamonkey-2.53.18.ebuild

```diff
@@ -0,0 +1,576 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
You should use **pkgdev commit** to handle things like this automatically :) it also adds a good commit message.